proto/rill/runtime/v1/api.proto

```diff
@@ -215,6 +223,25 @@ message PingResponse {
   google.protobuf.Timestamp time = 2;
 }
 
+message HealthRequest {}
```

Comment:
I like that `InstanceHealth` is separated out and it would be great to incorporate that into `rill project status`. But it would also be nice for the runtime-wide `Health` request to be able to check indexes – since a runtime isn't really healthy if its instances aren't, and it will make the health check easier to call (which is important to maximize it's utility).

We could potentially add a `bool skip_instances` option here in the `HealthRequest` for cases where the caller doesn't want the instance-level checks.

Reply:
Okay my understanding based on the previous comment was that we want to extract instance health checks from the health check API. I can add instance health checks as well.

Reply:
The `skip_instances` can be avoided as of now.